apis/kueue/v1beta1/workload_types.go

```diff
@@ -128,10 +128,13 @@ type PodSet struct {
 
 // WorkloadStatus defines the observed state of Workload
 type WorkloadStatus struct {
-	// admission holds the parameters of the admission of the workload by a
-	// ClusterQueue. admission can be set back to null, but its fields cannot be
+	// quotaReservation holds the parameters of the admission of the workload by a
+	// ClusterQueue. quotaReservation can be set back to null, but its fields cannot be
 	// changed once set.
-	Admission *Admission `json:"admission,omitempty"`
+	QuotaReservation *QuotaReservation `json:"quotaReservation,omitempty"`
+
+	// the same as quotaReservation, used for backwards compatibility
+	Admission *QuotaReservation `json:"admission,omitempty"`
```

Comment:
If find this rename (the API field change) unnecessary.

Admission is still a valid name: it contains the details about admission. It doesn't mean that the workload is admitted. That's what the condition is for.

Reply:
I can't say I agree ... however, done